package.json

```diff
@@ -54,7 +54,8 @@
   },
   "dependencies": {
     "sirv-cli": "^1.0.0",
-    "web-ext": "^5.1.0"
+    "web-ext": "^5.1.0",
+    "yarn": "^1.22.10"
```

Comment:
Did you mean to include yarn as a dep here?

Reply:
Thanks, yeah unfortunately the Taskcluster CI on mozilla-extensions/ org seems to prefer `yarn` and it's have some problem with the dev dependencies in our `package.json`. I suspect this is just a bug, we need to figure where to file this (and also the bug about the dep-signing job not working for forks, but that might be harder because of the security assumptions TC seems to be set up with).

Reply:
Using `yarn.lock` was suggested as a workaround, but I'd be really happy if we can just get rid of it.